ming_sim/beat_orchestration.py

```
@@ -348,3 +425,202 @@ def generate_close_beat_body(
+                bucket.append(self._executor.submit(_run, int(entry_id), inputs))
+
+    def start_open_enter(
+        self,
+        db: Any,
+        state: Any,
+        *,
+        minister_name: str,
+        chat_turn_id: int,
+        beat_generator: Optional[BeatGenerator],
+    ) -> None:
+        """锁内一次原子 claim：同 chat_turn_id 至多启动一轮 discover/submit。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)